src/libs/Navigation/types.ts

```diff
@@ -432,6 +426,29 @@ type MoneyRequestNavigatorParamList = {
         iouType: string;
         reportID: string;
     };
+    [SCREENS.MONEY_REQUEST.CREATE]: {
+        iouType: ValueOf<typeof CONST.IOU.TYPE>;
+        reportID: string;
+        transactionID: string;
+
+        // These are not used in the screen, but are needed for the navigation
+        // for IOURequestStepDistance and IOURequestStepAmount components
+        backTo: never;
+        action: never;
```

Comment:
I removed it locally and typecheck works fine, how about removing it?

Reply:
Hm, running `npm run typecheck` results in errors for me after removing it:
```
src/pages/iou/request/step/IOURequestStepAmount.tsx:47:52 - error TS2339: Property 'backTo' does not exist on type 'Readonly<{ iouType: ValueOf<{ readonly SEND: "send"; readonly SPLIT: "split"; readonly REQUEST: "request"; readonly TRACK_EXPENSE: "track-expense"; }>; reportID: string; transactionID: string; } | { ...; }>'.

47         params: {iouType, reportID, transactionID, backTo, action},
                                                      ~~~~~~

src/pages/iou/request/step/IOURequestStepAmount.tsx:47:60 - error TS2339: Property 'action' does not exist on type 'Readonly<{ iouType: ValueOf<{ readonly SEND: "send"; readonly SPLIT: "split"; readonly REQUEST: "request"; readonly TRACK_EXPENSE: "track-expense"; }>; reportID: string; transactionID: string; } | { ...; }>'.

47         params: {iouType, reportID, transactionID, backTo, action},
                                                              ~~~~~~

src/pages/iou/request/step/IOURequestStepDistance.tsx:51:18 - error TS2339: Property 'action' does not exist on type 'Readonly<{ iouType: ValueOf<{ readonly SEND: "send"; readonly SPLIT: "split"; readonly REQUEST: "request"; readonly TRACK_EXPENSE: "track-expense"; }>; reportID: string; transactionID: string; } | { ...; }>'.

51         params: {action, iouType, reportID, transactionID, backTo},
                    ~~~~~~

src/pages/iou/request/step/IOURequestStepDistance.tsx:51:60 - error TS2339: Property 'backTo' does not exist on type 'Readonly<{ iouType: ValueOf<{ readonly SEND: "send"; readonly SPLIT: "split"; readonly REQUEST: "request"; readonly TRACK_EXPENSE: "track-expense"; }>; reportID: string; transactionID: string; } | { ...; }>'.

51         params: {action, iouType, reportID, transactionID, backTo},
                                                              ~~~~~~


Found 4 errors in 2 files.

Errors  Files
     2  src/pages/iou/request/step/IOURequestStepAmount.tsx:47
     2  src/pages/iou/request/step/IOURequestStepDistance.tsx:51
```

Reply:
Okay, I have the same now. I think it's fine to leave `never` types here